### ISSUE TITLE
Running x9 csgo processes sandboxed

### ISSUE THREAD

**Issue by mashiro-shiina** [netblue30/firejail#1218]
I need to run x9 copies of csgo (from different steam accounts)
Means i need to run x9 copies of steam client
Is that possible with firejail?
Can you explain me how to make 9 "sandboxes" that would use same resources(from 1 folder) and run them in the same time? Is there any way to lower cpu load and save disk space by using same files for each sandbox?
Should i make custom profile for steam or default profile is ok?

**Comment by netblue30** [comment]
Quite a number of programs would not allow you to run two instances of the program in the same time. Among them you'll find firefox and chromium browsers. You can try to trick them into running multinstance, but it is specific to the program.

For Firefox is easy, you start one instance with "firejail firefox" and the second one "firejail --private firefox -no-remote". In order to do it you need to fake a new home directory (with firejail --private) and you also need some help from the program (-no-remote implemented by firefox).

So basically you are looking for some help from the program itself (steam client or csgo), fake home directories will be provided by firejail.
